### PR TITLE
feat: add no-animate rule

### PR DESCRIPTION
**Pull request by nemmip**
#### Pull Request purpose

[ ] Documentation update
[ ] Bug fix
[X] New rule
[X] Updates an existing rule
[ ] Add something to the core
[ ] Other, please explain:

#### Checklist

[X] Update CHANGELOG.md
[ ] Update task status in release plan (README)
[ ] Docs added (/docs/rules/<rule_name>)
[ ] Docs ready to be added to website (ec0lint.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,6 @@ Let's build an eco-friendly website!
 - :hammer: Plugin HTML (ec0lint-plugin-html)
 - :hammer: Lazy loading (require-lazy-loading, ec0lint-plugin-html)
 - :four_leaf_clover: Font source (no-hosted-online-fonts, ec0lint-style)
-- :ballot_box_with_check: Light libraries - jQuery andSelf (no-and-self, ec0lint)
 - :hammer: Light libraries – jQuery Animate (no-animate, ec0lint)
```
Comment (malwinq) [inline README.md]: Pls change the status to done

File: docs/rules/no-animate.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# no-animate
+
+Disallows the [`.animate`](https://api.jquery.com/animate/) method. Use the `allowScroll` option to allow animations which are just used for scrolling. Prefer CSS transitions.
+
+## Rule details
```
Comment (malwinq) [inline docs/rules/no-animate.md:11]: Do we have CO2 reduction here?

File: lib/rules/index.js [hunk 3]
```diff
@@ -14,9 +14,7 @@ module.exports = new LazyLoadingRuleMap(Object.entries({
     "lighter-http": () => require("./lighter-http"),
     "no-ajax": () => require("./no-ajax"),
     "no-ajax-events": () => require("./no-ajax-events"),
-    "no-and-self": () => require("./no-and-self"),
-    "no-ajax": () => require("./no-ajax"),
```
Comment (malwinq) [inline lib/rules/index.js]: How about no-ajax and no-ajax-events?

File: lib/rules/no-ajax.js [hunk 4]
```diff
@@ -5,14 +5,6 @@
...
 "use strict";
 
-//------------------------------------------------------------------------------
-// Requirements
-//------------------------------------------------------------------------------
-
-const astUtils = require('./utils/ast-utils');
-const ajaxTable = ['ajax', 'get', 'getJSON', 'getScript', 'post']
-
-
```
Comment (malwinq) [inline lib/rules/no-ajax.js:15]: Why is to not needed?

File: lib/rules/no-animate.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+							return;
+						}
+					}
+				}
+
+				if ( utils.isjQuery( context, node ) ) {
+					context.report( {
+						node: node,
+						message: allowScroll ?
+							'Prefer CSS transitions to .animate' :
```
Comment (malwinq) [inline lib/rules/no-animate.js]: We can rephrase it to be consistent with the form of other messages

File: lib/rules/no-ajax.js (hunk 4 above)
Comment (nemmip) [reply lib/rules/no-ajax.js:15]: Not used variables

File: README.md [hunk 6]
```diff
@@ -72,7 +72,7 @@ Let's build an eco-friendly website!
 - :ballot_box_with_check: Lazy loading (require-lazy-loading, ec0lint-plugin-html)
 - :hammer: Font source (no-hosted-online-fonts, ec0lint-style)
 - :ballot_box_with_check: Light libraries - jQuery andSelf (no-and-self, ec0lint)
-- :ballot_box_with_check: Light libraries – jQuery Animate (no-animate, ec0lint)
+- :white_check_mark: Light libraries – jQuery Animate (no-animate, ec0lint)
```
Comment (malwinq) [inline README.md]: let's leave the old icon

File: lib/rules/no-animate.js [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+module.exports = {
+	meta: {
+		type: 'suggestion',
+		docs: {
+			description:
+				'Disallows the ' + utils.jQueryCollectionLink( 'animate' ) +
+				' method. Use the `allowScroll` option to allow animations which are just used for scrolling. Prefer CSS transitions.',
+            category: "Code improvements",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/no-animate"
```
Comment (malwinq) [inline lib/rules/no-animate.js]: eslint -> ec0lint

**Review by malwinq — APPROVED** [review]
